finetrainers/trainer.py

```diff
@@ -341,6 +340,27 @@ def collate_fn(batch):
             pin_memory=self.args.pin_memory,
         )
 
+    def sort_out_weight_dtype(self, accelerator):
```

Comment:
For methods that are not going to be called via the `train.py` file, would prefer to have them as internal private and at the bottom of the class

```suggestion
    def _get_training_dtype(self, accelerator):
```